tasks/fix-cat2.yml

```diff
@@ -1166,15 +1166,20 @@
       - notimplemented
 
 - name: "MEDIUM | RHEL-07-030330 | PATCH | The operating system must immediately notify the System Administrator (SA) and Information System Security Officer ISSO (at a minimum) when allocated audit record storage volume reaches 75% of the repository maximum audit record storage capacity."
-  command: "true"
-  changed_when: no
+  lineinfile:
+    path: /etc/audit/auditd.conf
+    regexp: "space_left =.*"
+    line: "space_left = {{ rhel7stig_auditd_space_left }}"
+  register: rhel7stig_auditd_space_left_result
+  failed_when:
+    - rhel7stig_auditd_space_left_result | failed
+    - rhel7stig_auditd_space_left_result.rc != 257
```

Comment:
I think we only want to ignore the case of the file being absent when an absent file satisfies the STIG requirement.